### ISSUE TITLE
[Bug Report][2.6.14] Expansion Panel can get out of sync with its value/v-model

### ISSUE THREAD

**Issue by duncan-oxd** [vuetifyjs/vuetify#17032]
### Environment
**Vuetify Version:** 2.6.14
**Vue Version:** 2.7.10
**Browsers:** Chrome 111.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
If you click a `v-expansion-panel`, it will expand or collapse instantly, regardless of the state of the value prop on the `v-expansion-panels` component. If the state is set by something like an async operation, it can get out of sync.

The reproduction link has the v-model as a getter, so its value can't be changed, but the panels still open and close freely when you click.

### Expected Behavior
Panel state should always reflect the value of the props (only $emit a change event and not animate when you click)

### Actual Behavior
Panel state changes whenever it's clicked on

### Reproduction Link
[https://codepen.io/oxd_duncan/pen/QWVPoaz?editors=101](https://codepen.io/oxd_duncan/pen/QWVPoaz?editors=101)

### Other comments
A similar thing can happen with vuetify checkboxes, btw

**Comment by KaelWD** [comment]
You also get `[Vue warn]: Computed property "openPanels" was assigned to but it has no setter.`

v-model in v2 is always uncontrolled, the only way to have a static value is to watch it and set it back to what you want in a nextTick.

**Comment by duncan-oxd** [comment]
I was just using a static value to illustrate the bug, but the bug itself is more related to the fact that it changes state internally and then emits the new value, rather than emitting the value and only changing state when the prop changes.

In other words, I would expect it to use the value prop to determine its state, but clearly it's using something internally, which can get out of sync with the props. 

Maybe a more realistic example would be if the parent did something asynchronously and took a moment before the props updated. Let me know if that doesn't make sense.

**Comment by KaelWD** [comment]
Yeah it's been like that for years. We don't have any plans to change it in v2. v3 treats it differently if the corresponding update event is bound or not: https://github.com/vuetifyjs/vuetify/blob/docs/common-concepts/packages/docs/src/pages/en/concepts/v-model.md

**Comment by duncan-oxd** [comment]
I understand it would be a big overhaul for a v2 component, buthat's good news about v3. Thanks for the response 👍
